app/controllers/nonsipp/employercontributions/EmployerContributionsMemberListController.scala

```diff
@@ -62,26 +62,8 @@ class EmployerContributionsMemberListController @Inject()(
 
   def onPageLoad(srn: Srn, page: Int, mode: Mode): Action[AnyContent] = identifyAndRequireData(srn) {
     implicit request =>
-      val memberMap = request.userAnswers.map(MembersDetailsPages(srn))
```

Comment:
Tagging @ccatosdevelopment for awareness, please check if this is still ok with your changes going on in parallel with this change

Reply:
Thanks @kberec, this doesn't overlap with any of the changes I've made so far, but I'll keep it in mind if I make any further changes to this file.